main.go

```diff
@@ -96,6 +96,8 @@ func getLog(
 	// intentions...
 	writer.Header().Set("Content-Type", "text/plain; charset=utf-8")
 	writer.Header().Set("Access-Control-Allow-Origin", "*")
+	writer.Header().Set("Transfer-Encoding", "chunked")
+	writer.Header().Set("Access-Control-Expose-Headers", "Transfer-Encoding")
```

Comment:
@petemoore, we definitely still need this before eli can use the transfer-encoding header in cors requests.